examples/utils/bundle-size-tests/.eslintrc.cjs

```diff
@@ -7,6 +7,8 @@ module.exports = {
 	extends: [
 		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
 		"prettier",
+		// There are a lot of intentional internal APIs leveraged here for simplicity. Skip common example rules:
+		// "../../.eslintrc.cjs",
```

Comment:
I don't like tools though for this. We can create dedicated test space. If we renamed examples to tests, this would fit into place but still have internal use at odds with other tests.

Reply:
It's not really about tests either though, it's just a tool we use in our calculation of bundle sizes. It doesn't _test_ anything.